src/pubsub/peer-streams.js

```diff
@@ -17,20 +20,22 @@ log.error = debug('libp2p-pubsub:peer-streams:error')
  * @param {Uint8Array} source
  * @returns {Promise<Uint8Array>}
  *
- * @typedef {object} DuplexIterableStream
- * @property {Sink} sink
- * @property {() AsyncIterator<Uint8Array>} source
+ * @typedef {object} MuxedStream
+ * @type import('../stream-muxer/types').MuxedStream
  *
  * @typedef PeerId
  * @type import('peer-id')
+ *
+ * @typedef PushableStream
+ * @type import('it-pushable').Pushable<Uint8Array>
```

Comment:
can you fix this syntax 
`@typedef {import('../stream-muxer/types').MuxedStream} MuxedStream`